lib/developer_portal/app/controllers/developer_portal/application_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 module DeveloperPortal
   class ApplicationController < ::FrontendController
+    include ::CMS::Toolbar
```

Comment:
I think this is the right place. I moved it from `app/controllers/frontend_controller.rb`. Previously, the CMS token was being checked for all URLs under admin and developer portals, now only for developer portal.

Reply:
Apparently it wasn't because some tests failed. So in fact, the CMS token was and is still being checked on all request to admin portal 🙃